poseidon-tornado/README.md

```diff
@@ -1,15 +0,0 @@
-# Poseidon-Tornado wrapper
```

Comment:
We want this soon, but I agree it is in a so-so state and better to base based on RLN from scratch at this point.

Reply:
Removed in https://github.com/vacp2p/zerokit/commit/1d8a5f59b60eea88cdd69d68df760ebb3ae73e54

cc @staheri14 if you want to see what this code is doing